src/templates/open-enterprise/index.js

```diff
@@ -0,0 +1,152 @@
+  sourceCodeUrl: 'https://github.com/AutarkLabs/open-enterprise',
+  registry: 'aragonpm.eth',
+  apps: [
+    { appName: 'voting.aragonpm.eth', label: 'Voting' },
+    { appName: 'token-manager.aragonpm.eth', label: 'Tokens' },
+    { appName: 'finance.aragonpm.eth', label: 'Finance' },
+    { appName: 'address-book.aragonpm.eth', label: 'Address Book' },
+    { appName: 'allocations.aragonpm.eth', label: 'Allocations' },
+    { appName: 'dot-voting.aragonpm.eth', label: 'Dot Voting' },
+    { appName: 'projects.aragonpm.eth', label: 'Projects' },
+    { appName: 'rewards.aragonpm.eth', label: 'Rewards' },
+  ],
+  screens: [
+    [
+      data => completeDomain(data.domain) || 'Claim domain',
+      props => <ClaimDomainScreen screenProps={props} />,
+    ],
+    ['Configure template', props => <VotingScreen screenProps={props} />],
+    ['Configure template', props => <DotVotingScreen screenProps={props} />],
+    ['Configure template', props => <TokensScreen screenProps={props} />],
+    [
+      'Review information',
+      props => {
+        const { domain, dotVoting, tokens, voting } = props.data
+        return (
+          <ReviewScreen
+            screenProps={props}
+            items={[
+              {
+                label: 'General info',
+                fields: [
+                  ['Organization template', 'Open Enterprise'],
+                  ['Name', completeDomain(domain)],
+                ],
+              },
+              {
+                label: (
+                  <KnownAppBadge appName="voting.aragonpm.eth" label="Voting" />
+                ),
+                fields: VotingScreen.formatReviewFields(voting),
+              },
+              {
+                label: (
+                  <KnownAppBadge
+                    appName="dot-voting.aragonpm.eth"
+                    label="Dot Voting"
+                  />
+                ),
+                fields: DotVotingScreen.formatReviewFields(dotVoting),
+              },
+              {
+                label: (
+                  <KnownAppBadge
+                    appName="token-manager.aragonpm.eth"
+                    label="Tokens"
+                  />
+                ),
+                fields: TokensScreen.formatReviewFields(tokens),
+              },
+            ]}
+          />
+        )
+      },
+    ],
+  ],
+  prepareTransactions(createTx, data) {
+    const allocationsPeriod = 0 // default
+    const financePeriod = 0 // default
+
+    const { domain, dotVoting, tokens, voting } = data
+    const { tokenName, tokenSymbol, members } = tokens
+
+    const baseStake = new BN(10).pow(new BN(18))
+    const stakes = members.map(([_, stake]) =>
+      baseStake.mul(new BN(stake.toString())).toString()
+    )
+    const accounts = members.map(([account]) => account)
+
+    const { support, quorum, duration } = voting
+    const onePercent = new BN(10).pow(new BN(16))
+    const adjustedSupport = onePercent.mul(new BN(support)).toString()
```

Comment:
Let's use [`adjustVotingSettings()`](https://github.com/aragon/aragon/blob/master/src/templates/company/index.js#L20) for this, like the other templates, because voting reverts if either percentage is 100%